velox/exec/HashPartitionFunction.h

```diff
@@ -37,8 +37,20 @@ class HashBitRange {
     return (hash >> begin_) & fieldMask_;
   }
 
+  uint8_t startBitOffset() const {
```

Comment:
Would it make sense to rename these new methods to begin() and end() to match the constructor?

Reply:
Make sense!